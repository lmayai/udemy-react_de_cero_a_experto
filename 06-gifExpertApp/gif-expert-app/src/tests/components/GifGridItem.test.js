import React from 'react';
import { shallow } from 'enzyme';
import { GifGridItem } from '../../components/GifGridItem';
import '@testing-library/jest-dom';

describe('Testing on GifGridItem', () => {
  test('should render gif grid item', () => {
    const wrapper = shallow(<GifGridItem />);
    expect(wrapper).toMatchSnapshot();
  })
  
})
