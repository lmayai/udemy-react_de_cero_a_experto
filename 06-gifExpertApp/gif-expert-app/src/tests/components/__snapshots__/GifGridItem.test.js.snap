// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing on GifGridItem should render gif grid item 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img />
  <p />
</div>
`;
